### DISCUSSION TITLE
Persistence Storage GUI app

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1366] [Ideas]
It would be very user friendly to have a Persistent Storage GUI app or to be able to generate persistent storage files in the Ventoy GUI app.

**Comment by ventoy** [comment]
Persistence is not used by every Ventoy user, so a shell script is enough and more flexible.

**Reply by trymeouteh** [reply]
Is there another app on Linux that can generate persistent storage files?

**Reply by ventoy** [reply]
No
